### PR TITLE
Bussin has multi-line strings. Updated vars & funcs for bussin. Updated java annotations.

### PR DESCRIPTION
**Pull request by Symmettry**

### CODE REVIEW

File: Lua/Plugins/bsx.lua [hunk 1]
```diff
@@ -82,6 +88,9 @@ function detect_functions(content)
         "format",
         "fetch",
         "len",
+        "import",
+        "splitstr",
```
Comment (RealPacket) [inline Lua/Plugins/bsx.lua:92]: You can't develop Bussin X plugins... Right?

File: Lua/Plugins/bsx.lua [hunk 2]
```diff
@@ -51,7 +53,11 @@ function detect_variables(content)
     local variable_names = {
         "error",
         "nerd",
-        "fs"
+        "fs",
```
Comment (RealPacket) [inline Lua/Plugins/bsx.lua:56]: I don't remember if you can use the File System in Bussin X or Bussin.

**Review by RealPacket — COMMENTED** [review]
Not sure about if Bussin has some globals assumed to be in this PR,
such as `trim` or `splitstr`.

**Comment by Symmettry** [comment]
trim() and splitstr() are in my current bussin pr

i added file system to bussin a bit ago
same with import()

**Comment by RealPacket** [comment]
> trim() and splitstr() are in my current bussin pr
> 
> i added file system to bussin a bit ago same with import()
> 
> I included them here since I didn't want to have make a new pr for this to update it

Oh, OK.

**Review by RealPacket — APPROVED** [review]
Looks good to me.

**Pull request merged**